### PR TITLE
Add P1 footprint

### PR DESCRIPTION
**Pull request by emcniece**
Adding P1 PCB footprint, generated schematic components for Core, Photon, Electron, P1.

### CODE REVIEW

**Comment by rnestler** [comment]
Thanks for the Pull-Request! We will try to take a look at it in the next weeks. Unfortunately a lot of our members are currently on holidays.

**Comment by dbrgn** [comment]
Thanks! Unfortunately I can't test this right now because the KiCAD footprint editor segfaults :/

Regarding the schematic components, I'm not sure how KiCAD handles this properly. The `.pretty` directory is in itself [already a footprint library](http://kicad-pcb.org/help/file-formats/). Schematic symbols should probably go into a separate repository, right? Is github integration only implemented for footprints so far? Do you know more about this, @emcniece?

KiCAD is a mess :( Let's hope @ubruhin and @rnestler manage to finish a first version of http://librepcb.org/ soon :)

File: README.md [hunk 1]
```diff
@@ -7,10 +7,11 @@ KiCad footprints for Particle hardware (e.g. Core, Photon, Electron).
 - [Core](https://docs.particle.io/datasheets/core-datasheet/)
 - [Photon](https://docs.particle.io/datasheets/photon-datasheet/)
 - [Electron](https://docs.particle.io/datasheets/electron-datasheet/)
+- [P1](https://docs.particle.io/datasheets/p1-datasheet/)
 
 ## Missing Footprints
 
-- [P1](https://docs.particle.io/datasheets/p1-datasheet/)
+- [P0](https://docs.particle.io/datasheets/p1-datasheet/)
```
Comment (dbrgn) [inline README.md:14]: I can't find anything about the P0. Do you have more info?

**Comment by dbrgn** [comment]
@emcniece I created a separate repo for the schematics library at https://github.com/coredump-ch/particle.lib. Would you mind moving your schematics changes to a separate PR on that repo? Once it's there we can cross-link the two repos.

**Comment by dbrgn** [comment]
This can be closed as #2 has been merged.

**Pull request closed without merging**